src_testbed/lines/render_dim.rs

```diff
@@ -54,17 +55,17 @@ pub mod r3d {
             //use VertexFormat::{Float32x3, Float32x4};
 
             let mut shader_defs = Vec::new();
-            shader_defs.push("LINES_3D".to_string());
+            shader_defs.push("LINES_3D".to_string().into());
             if depth_test {
-                shader_defs.push("DEPTH_TEST_ENABLED".to_string());
+                shader_defs.push("DEPTH_TEST_ENABLED".to_string().into());
             }
 
             let vertex_buffer_layout = layout.get_layout(&[
                 Mesh::ATTRIBUTE_POSITION.at_shader_location(0),
                 Mesh::ATTRIBUTE_COLOR.at_shader_location(1),
             ])?;
             let (label, blend, depth_write_enabled);
-            if key.contains(MeshPipelineKey::TRANSPARENT_MAIN_PASS) {
+            if key.contains(MeshPipelineKey::BLEND_ALPHA) {
```

Comment:
is this correct? i don't know how this system works or what the enums mean. this is also not mentioned in the bevy upgrade notes 0.9-0.10-0.11